### PR TITLE
Update the election process timelines to new

### PR DESCRIPTION
**Pull request by kalyan678**
I've made changes to the election process timelines, updating them to reflect the new schedule compared to previous dates.

### CODE REVIEW

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Comment by tkoyama010** [comment]
@all-contributors please ad @kalyan678 for doc

**Comment by aterrel** [comment]
According to this pr the voting period has already happened. Is that the case? I have not seen such an announcement.

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Oops, we have to fix that. Thanks for pointing it out.

**Comment by tkoyama010** [comment]
@all-contributors please add @kalyan678 for doc

**Comment by tkoyama010** [comment]
@all-contributors please add @aterrel for review

File: README.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ The former group deals with higher-level strategic and administrative decisions,
...
-| Review election results (by NumFOCUS board)                           | Aug 9–Aug 23                  |
-| Publish results announcing new board members                          | Late August / Early September |
+| Election phases                                                       | Tentative dates                |
+| --------------------------------------------------------------------- | ------------------------------ |
+| Nomination period                                                     | June 14–July 5                 |
+| Nominee application review (by election committee and NumFOCUS board) | July 5–July 12                 |
+| Notify nominees and get confirmation                                  | Aug 12–Aug 21                  |
+| Voting period                                                         | Aug 21– Sep 9                  |
+| Review election results (by NumFOCUS board)                           | Sep 9–Sep 23                   |
+| Publish results announcing new board members                          | Late September / Early October |
```
Comment (tkoyama010) [inline README.md:47]:
```suggestion
| Notify nominees and get confirmation                                  | July 12–Sep 23                  |
| Voting period                                                         | Sep 23– Oct 7                  |
| Review election results (by NumFOCUS board)                           | Oct 7–Oct 23                   |
| Publish results announcing new board members                          | Late October / Early November |
```

File: README_ja.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ NumFOCUS の運営委員会は、運営理事会と技術運営委員会で構
...
-| 選挙結果の確認 (NumFOCUS理事会による)                   | 8月9日 - 8月23日  |
-| 新理事会メンバーを発表する結果を公表                    | 8月下旬 - 9月上旬 |
+| 選挙フェーズ                                            | 暫定的な日程       |
+| ------------------------------------------------------- | ------------------ |
+| 推薦期間                                                | 6月14日 - 7月5日   |
+| 候補者の申請審査 (選挙管理委員会とNumFOCUS理事会による) | 7月5日 - 6月12日   |
+| 候補者に通知し、確認を取る                              | 8月12日 - 8月26日  |
+| 投票期間                                                | 8月26日 - 9月9日   |
+| 選挙結果の確認 (NumFOCUS理事会による)                   | 9月9日 - 9月23日   |
+| 新理事会メンバーを発表する結果を公表                    | 9月下旬 - 10月上旬 |
```
Comment (tkoyama010) [inline README_ja.md:45]:
```suggestion
| 候補者に通知し、確認を取る                              | 6月12日 - 9月23日  |
| 投票期間                                                | 9月23日 - 10月7日   |
| 選挙結果の確認 (NumFOCUS理事会による)                   | 10月7日 - 10月23日   |
| 新理事会メンバーを発表する結果を公表                    | 10月下旬 - 11月上旬 |
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
@pavithraes @lsilen How about this schedule?

File: README.md (hunk 1 above)
Comment (aterrel) [reply README.md:47]: okay, do you need any support from the infrastructure team on the vote rollout?

**Review by aterrel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pavithraes) [reply README.md:47]: @tkoyama010 @kalyan678 - I think we can also mention the voter registration phase in the timeline. So, 
* Voter registration: 23rd Sep to 4th Oct 
* Voting period: 7th Oct - 25th Oct
* Review election results: 26th Oct - 8th Nov
* Publish results: mid Nov

Comment (tkoyama010) [reply README.md:47]:
```suggestion
| Voter registration                                   | Sep 20–Oct 4                  |
| Voting period                                                         | Oct 7– Oct 25                  |
| Review election results (by NumFOCUS board)                           | Oct 26–Nov 8                   |
| Publish results announcing new board members                          | Middle Nov |
```

File: README_ja.md (hunk 2 above)
Comment (tkoyama010) [reply README_ja.md:45]:
```suggestion
| 候補者に通知し、確認を取る                              | 9月20日 - 10月4日  |
| 投票期間                                                | 10月7日 - 10月25日   |
| 選挙結果の確認 (NumFOCUS理事会による)                   | 10月26日 - 11月8日   |
| 新理事会メンバーを発表する結果を公表                    | 11月中旬 |
```

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Review by pavithraes — APPROVED** [review]
Thanks, looks good to me!

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Comment by tkoyama010** [comment]
@all-contributors please add @pavithraes for projectManagement

**Pull request merged**